nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -111,6 +108,10 @@ def hw_config(self) -> HWConfig:
     def quant_trait_op_dict(self) -> Dict[int, OperatorMetatype]:
         return DEFAULT_ONNX_QUANT_TRAIT_TO_OP_DICT
 
+    @property
+    def reducer_map(self) -> Dict[StatisticsType, TensorReducerBase]:
+        return REDUCERS_MAP
+
```

Comment:
Please don't forget to introduce the abstract `reducer_map` property in the `MinMaxAlgoBackend`

Reply:
Yep!